scene/theme/default_theme.cpp

```diff
@@ -360,6 +360,27 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	theme->set_constant("check_v_offset", "CheckButton", 0);
 	theme->set_constant("outline_size", "CheckButton", 0);
 
+	// Button variations
+
+	theme->set_type_variation("FlatButton", "Button");
+	theme->set_type_variation("FlatMenuButton", "MenuButton");
+
+	Ref<StyleBoxFlat> flat_button_normal = button_normal->duplicate();
+	flat_button_normal->set_bg_color(Color(0, 0, 0, 0));
+	flat_button_normal->set_border_color(Color(0, 0, 0, 0));
```

Comment:
You could make StyleboxEmpty and copy margins/border size from the normal stylebox.

Reply:
Yes, I decided to do it this weird way because it would result in less upkeep if changes happen. Not that our theme files are well organized 🙁

Reply:
I mean, you can just do something like
```C++
for (int i = 0; i < 4; i++) {
	empty_box->set_content_margin(i, button_normal->get_margin(i) + button_normal->get_border_width(i))
}
```

Reply:
Sure, I can do that.